managed/services/agents/mongodb.go

```diff
@@ -252,6 +257,11 @@ func defaultCollectors(collectAll bool) map[string]collectorArgs {
 			enabled:     collectAll,
 			enableParam: "--collector.topmetrics",
 		},
+		// disabled until we have better information on the resources usage impact
```

Comment:
Is this comment correct? Why we are saying that it's disables when it's configured by `collectAll` parameter in this func?

Reply:
collectAll is by default disabled, but yeah since it is parameter in func it is not fully true. This occurs on more places. Removed from all places where you can pass this as a param.